tox.ini

```diff
@@ -238,6 +263,46 @@ commands =
     coverage report --ignore-errors --skip-covered --omit "wandb/vendor/*"
     codecov -e TOXENV -F unittest
 
+# temporary until we can consolidate coverage collection paths
```

Comment:
what do you have in mind for that?

Reply:
there is very subtle differences between the different tox environments... tox allows you to parameterize things in smarter way so we wont have to duplicate so much of the toxenv definitions.